src/views/forms/AccountManagementForms/WithdrawForm.tsx

```diff
@@ -222,16 +253,21 @@ export const WithdrawForm = () => {
     },
   ];
 
+  const { sanctionedAddresses } = useRestrictions();
+
   const errorMessage = useMemo(() => {
     if (error) {
-      return error?.message
-        ? stringGetter({
-            key: STRING_KEYS.SOMETHING_WENT_WRONG_WITH_MESSAGE,
-            params: { ERROR_MESSAGE: error.message },
-          })
-        : stringGetter({ key: STRING_KEYS.SOMETHING_WENT_WRONG });
+      return stringGetter({
+        key: STRING_KEYS.SOMETHING_WENT_WRONG_WITH_MESSAGE,
+        params: { ERROR_MESSAGE: error },
+      });
     }
 
+    if (toAddress && sanctionedAddresses.has(toAddress))
+      return stringGetter({
+        key: STRING_KEYS.TRANSFER_INVALID_DYDX_ADDRESS,
+      });
+
     if (!toAddress) return stringGetter({ key: STRING_KEYS.WITHDRAW_MUST_SPECIFY_ADDRESS });
```

Comment:
super nit, move the `if (!toAddress)` first so the later one only needs to check `if (sanctionedAddresses.has(toAddress))`?